www/_template/guides/svelte.md

```diff
@@ -1,12 +1,294 @@
+
+```diff
+  mount: {
+-   /* ... */
++   // directory name: 'build directory'
++   public: '/',
++   src: '/_dist_',
+  },
+```
+
+<img src="/img/guides/react/folderstructure.png" alt="The original file configuration had Snowpack building the directory structure the same as the directories in the project, including root. Now the config builds only src and public. Src to the dist folder and public to root." />
+
+`mount` is part of the [Snowpack Configuration API](https://www.snowpack.dev/#configuration). It allows you to customize the file structure of your project. The key is the name of the directory and the value is where you'd like them in the final build. With this new configuration, Snowpack builds files in `public` like `public/index.css` directory into `index.css`. It builds files in `src` like `src/index.js` into `/_dist_/index.js`, so you'll need to change that path in your `index.html`:
+
+```diff
+  <body>
+-   <h1>Welcome to Snowpack!</h1>
+-   <script type="module" src="/index.js"></script>
++   <script type="module" src="/_dist_/index.js"></script>
+  </body>
+```
+
+You'll need to restart Snowpack for configuration file changes. When you start up again, if it worked, it should look the same.
+
+## Styling your project
+
+Your site is a little plain, so in this step you'll add an image and animate it with CSS.
+
+> 💡 Tip: as you're doing this, you should not need to reload the page or restart Snowpack. Snowpack automatically updates the project in the browser as you edit code.
+
+Add this file [`logo.svg`](https://github.com/snowpackjs/snowpack/blob/master/create-snowpack-app/app-template-svelte/public/logo.svg) to your `public` directory. Now you can add it to your `App.svelte`
+
+```diff
+    <header class="App-header">
++       <img src="/logo.svg" class="App-logo" alt="logo" />
+        <a class="App-link" href="https://svelte.dev" target="_blank" rel="noopener noreferrer">
+          Learn Svelte
+```
+
+Image: Site showing the new logo
+
+With Svelte, CSS can go directly in your `.svelte` component. Add this code to the top of `App.svelte` between the `<style>` tags:
+
+```css
+<style>
+  .App-header {
+    background-color: #f9f6f6;
+    color: #333;
+    min-height: 100vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    font-size: calc(10px + 2vmin);
+  }
+  .App-logo {
+    height: 36vmin;
+    pointer-events: none;
+    margin-bottom: 3rem;
+    animation: App-logo-spin infinite 1.6s ease-in-out alternate;
+  }
+  @keyframes App-logo-spin {
+    from {
+      transform: scale(1);
+    }
+    to {
+      transform: scale(1.06);
+    }
+  }
+</style>
+```
+
+> 💡 Tip: Snowpack has built-in support for [CSS Modules](https://www.snowpack.dev/#import-css-modules) and if you'd like to use Sass there is an official [Sass Plugin](https://www.snowpack.dev/#sass).
+
+Image: Gif showing the results with the logo animated
+
+## Adding a simple counter
```

Comment:
maybe "going further" or "expanding your component"? I could go either way, just wanted to bring it up